astropy/wcs/wcs.py

```diff
@@ -129,6 +130,11 @@
 # Additional relax bit flags
 WCSHDO_SIP = 0x10000
 
+# Regular expression defining SIP keyword
+# It matches keyword that starts with A, B, AP or B, followed by an underscore then
+# a number in range of 0-19, followed by an underscore and another number
+# in range of 0-19. Keyword optionally ends with a capital letter.
```

Comment:
According to @nden's comment, the optional last letter can only be "L" to indicate an alternate WCS, or am I confused?